hpat/tests/test_series.py

```diff
@@ -499,10 +529,15 @@ def test_impl(A, B):
         B = pd.Series(np.ones(n))
         np.testing.assert_array_equal(hpat_func(A.copy(), B), test_impl(A, B))
 
-    @unittest.skip('AssertionError - fix needed\n'
-                   'Attribute "dtype" are different\n'
-                   '[left]:  int64\n'
-                   '[right]: int32\n')
+    def test_series_abs(self):
+        def test_impl(A):
+            return A.abs()
+        hpat_func = hpat.jit(test_impl)
+
+        S = pd.Series([1, 2.2, -5.8, 3, 2, -1.2, -4, 4.7])
```

Comment:
Does it supports bin, oct and hex numbers? Scientific notation (0.5E-01)?

Reply:
@shssf I overlooked it, there's no need in this test at all - there is another one test_series_abs1, which also checks NaN. Anyway, it works, yes. I will add hex/dec/oct literals to it.